tests/test_registrations/test_embargoes.py

```diff
@@ -903,7 +903,7 @@ def test_POST_register_embargo_is_not_public(self, mock_enqueue):
 
         assert_equal(res.status_code, 202)
 
-        registration = Registration.find().order_by('-registered_date').first()
+        registration = Registration.objects.all().order_by('-registered_date').first()
```

Comment:
Similar to the FYI note above, `.all()` isn't needed here (but doesn't hurt anything either).